components/image-builder-mk3/README.md

```diff
@@ -72,3 +72,35 @@ To run all `image-builder-mk3` tests:
 ```console
 go test -v ./...
 ```
+
+# Required Permssion
+
+If you want it to work in a particular public cloud, you may need to grant some permissions.
+Below is a reference for this.
+
+## AWS
+
+If you would like to use ECR as a container registry, please add the following IAM policy below.
+
+```json
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Sid": "VisualEditor0",
+            "Effect": "Allow",
+            "Action": [
+                "ecr-public:BatchGetImage",
+                "ecr-public:GetAuthorizationToken",
+                "ecr-public:BatchCheckLayerAvailability",
+                "ecr-public:PutImage",
```

Comment:
I thought the image-builder-mk3 did not require the PutImage policy. Only image-imager-bob requires.
Am I wrong?

Reply:
I thought about that a bit too, but felt it would be impractical to use something different than image-builder-bob.